### ISSUE TITLE
[bug] Dark theme not working correctly on iOS

### ISSUE THREAD

**Issue by META-DREAMER** [tamagui/tamagui#298]
**Describe the bug**
Trying to setup the example app with dark theme on native iOS, but only some elements are updating their colors. Works fine on web.

Light Theme         |  Dark Theme
:-------------------------:|:-------------------------:
![simulator_screenshot_A108A1F6-433F-49BC-944D-4EF2BB0E02C7](https://user-images.githubusercontent.com/7143583/197310123-9eecab72-f7b4-4e68-9857-85ea2faa0a49.png)  |  ![simulator_screenshot_5DFE2C08-903D-4596-B8B9-5527B3F09ACB](https://user-images.githubusercontent.com/7143583/197310130-613cb95f-f8ea-413b-9f5c-24a8e4ff8576.png)

Light on Web         |  Dark on Web
:-------------------------:|:-------------------------:
  ![image](https://user-images.githubusercontent.com/7143583/197310218-323dbdc5-dbe3-42ce-9c97-bbc2e2ca7161.png) |  ![image](https://user-images.githubusercontent.com/7143583/197310182-971a37b5-e6f2-43db-9d82-3c39d22be813.png)

**To Reproduce**

Repro repo: https://github.com/META-DREAMER/tamagui-app

**System Info (please complete the following information):** 

```

  System:
    OS: macOS 12.6
    CPU: (10) x64 Apple M1 Max
    Memory: 64.89 MB / 64.00 GB
    Shell: 3.3.1 - /opt/homebrew/bin/fish
  Binaries:
    Node: 14.18.2 - /var/folders/r1/c3ctl7_s2dl2wld20w1mgj7m0000gn/T/fnm_multishells/64611_1666398493594/bin/node
    Yarn: 3.2.3 - /opt/homebrew/bin/yarn
    npm: 8.3.0 - /var/folders/r1/c3ctl7_s2dl2wld20w1mgj7m0000gn/T/fnm_multishells/64611_1666398493594/bin/npm
    Watchman: 2022.10.17.00 - /opt/homebrew/bin/watchman
  Browsers:
    Brave Browser: 104.1.42.97
    Firefox: 103.0.2
    Safari: 16.0
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.18.9 
    @manypkg/cli: ^0.19.1 => 0.19.1 
    @types/react: ^18.0.15 => 17.0.21 
    @types/react-native: ^0.69.5 => 0.69.5 
    eslint: ^8.21.0 => 8.21.0 
    prettier: ^2.7.1 => 2.7.1 
    turbo: ^1.4.2 => 1.4.2 
    typescript: ^4.7.4 => 4.7.4 

```

**Tamagui info**

check repro repo

**Comment by bang88** [comment]
@META-DREAMER   I have the same question. Can you manually change the simulator to dark mode and the app will use the dark theme.  But it will not run in dark mode automatically.

**Comment by natew** [comment]
I was seeing a few problems due to the compiler that look like this sometimes, perhaps a recent beta caused. But the next release fixes all of it. It's very close, out this weekend.

**Comment by dohomi** [comment]
@META-DREAMER try to specify a size - this was fixing it in my environment.

**Comment by META-DREAMER** [comment]
> I was seeing a few problems due to the compiler that look like this sometimes, perhaps a recent beta caused. But the next release fixes all of it. It's very close, out this weekend.

any update on this? still not working as of beta.201

**Comment by natew** [comment]
Fixed in beta 204 via 3d69c3dde8791d3c1b1346abc4916d69881508ec, sorry about that.
